var/spack/repos/builtin/packages/kadath/package.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+
+
+class Kadath(CMakePackage):
+    """KADATH SPECTRAL SOLVER
+
+    The Frankfurt University/Kadath (FUKA) Initial Data solver branch is
+    a collection of ID solvers aimed at delivering consistent initial
+    data (ID) solutions to the eXtended Conformal Thin-Sandwich (XCTS)
+    formulation of Einstein's field equations for a variety of compact
+    object configurations to include extremely compact, asymmetric, and
+    mixed spin binaries.
+    """
+
+    homepage = "https://kadath.obspm.fr/fuka/"
+    url      = "https://gitlab.obspm.fr/grandcle/Kadath.git"
+
+    maintainers = ['eschnett']
+
+    version('fuka', git="https://gitlab.obspm.fr/grandcle/Kadath.git", branch='fuka')
+
+    variant('mpi', default=True)
+
+    variant('codes', multi=True,
+            description="Codes to enable",
+            values=('BBH', 'BH', 'BHNS', 'BNS', 'NS'),
+            default=())
+
+    depends_on('blas')
+    depends_on('boost cxxstd=17')         # kadath uses std=C++17
+    depends_on('cmake @2.8:', type='build')
+    depends_on('fftw @3:')
```

Comment:
```suggestion
    depends_on('fftw-api @3:')
```
We have a virtual provider for this